core/config.class.inc.php

```diff
@@ -113,70 +113,86 @@ class Config
 	 * @since 2.7.0 export_pdf_font param
 	 */
 	protected $m_aSettings = [
-		'log_level_min' => [
-			'type' => 'array',
-			'description' => 'Optional min log level, per channel.',
-			'default' => '',
-			'value' => '',
-			'source_of_value' => '',
+		'log_level_min'                      => [
+			'type'                => 'array',
+			'description'         => 'Optional min log level, per channel.',
+			'default'             => '',
+			'value'               => '',
+			'source_of_value'     => '',
 			'show_in_conf_sample' => false,
 		],
-		'log_level_min.write_in_db' => [
-			'type' => 'array',
-			'description' => 'Optional min log level IN DB, per channel.',
-			'default' => '',
-			'value' => '',
-			'source_of_value' => '',
+		'log_level_min.write_in_db'          => [
+			'type'                => 'array',
+			'description'         => 'Optional min log level IN DB, per channel.',
+			'default'             => '',
+			'value'               => '',
+			'source_of_value'     => '',
 			'show_in_conf_sample' => false,
 		],
-		'event_service.debug.filter_events' => [
-			'type' => 'array',
-			'description' => 'List of events name to filter Event Service debug messages',
-			'default' => [],
-			'value' => '',
-			'source_of_value' => '',
+		'log_purge_enable'                   => [
```

Comment:
```suggestion
		'log_purge_enabled'                   => [
```

Preterit form is used in all params, I would like to keep the consistency.